vigilance/default_suites/cobertura.py

```diff
@@ -75,7 +75,7 @@ def _get_attribute(xml, attr, default=0):
         try:
             return float(xml.attrib[attr]) * 100
         except (KeyError, ValueError):
-            logging.getLogger(__name__).warn('Failed to find attribute in XML element: %s', attr)
+            logging.getLogger(__name__).warning('Failed to find attribute in XML element: %s', attr)
```

Comment:
was this broken before or is there a compatibilty issue with py2 vs py3

Reply:
Wasn't broken, but py3 was giving a warning about warn() being deprecated. But warning() exists in both versions.